internal/kms/kms_test.go

```diff
@@ -192,7 +192,6 @@ func Test_ListKeys(t *testing.T) {
 }
 
 func Test_RotateKeys(t *testing.T) {
-	t.Parallel()
```

Comment:
I see what's happening here, we're using the same `kmsCache` between multiple concurrently running tests. We can leave this one in place, but we should either give each subtest its own "kmsCache", or remove t.Parallel() from inside each subtest.

Reply:
I tried doing the `kmsCache` route, but was getting failures/doing something wrong. Pushed up a change for doing the `t.Parallel` route.